builder/lxc/config.go

```diff
@@ -65,6 +73,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		errs = packer.MultiErrorAppend(errs, fmt.Errorf("Failed parsing init_timeout: %s", err))
 	}
 
+	if c.LxcTemplate.Name != "" && c.RootFs != (RootFsConfig{}) {
+		errs = packer.MultiErrorAppend(errs, fmt.Errorf("Cannot build with both lxc_template and rootfs configuration options"))
```

Comment:
I wonder if we could clean up some of the later code in creation by using the `none` template. I'm not sure if it does more than just create an empty directory in the lxc path though.